README.md

```diff
@@ -1,11 +1,54 @@
-## My Project
+## ACK Multi-Controller Helm Chart
 
-TODO: Fill this README out!
+The ACK Multi-Controller Helm chart, shortened to `ack-chart`, acts as a wrapper
+for all ACK controllers released under General Availability (GA). 
 
-Be sure to:
+`ack-chart` lists each ACK controller as a subchart dependency.
 
-* Change the title in this README
-* Edit your repository description on GitHub
+`ack-chart` is currently in **developer preview** and is subject to breaking
```

Comment:
This important note deserves its own heading (e.g., `Release Status` or something) or at least to be put in all bold for now.